ocean/coupling_files_E3SM/create_E3SM_MPAS_coupling_files.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+"""
+This script creates coupling files needed to run an MPAS-Ocean and
+MPAS-seaice within E3SM.
+"""
+import os
```

Comment:
To support both python 2 and python 3, it  would be good to add (as the first import):
```suggestion
from __future__ import absolute_import, division, print_function, \
    unicode_literals
    
import os
```